python/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  notebooks:
+    build: .
+    ports:
+      - 8888:8888
+    volumes:
+      - $PWD/notebooks:/home/jovyan/work
```

Comment:
Not sure what this is?

Reply:
- notebooks are hosted at localhost:8888
- the `jupyter/scipy-notebook` image automatically sets up a directory for notebooks at `/home/jovyan/work` (not sure who `jovyan` is!). I'm just passing the contents of `$PWD/notebooks` over there so that the user can read/write the example notebooks